extensions/scheduler/deployment/src/test/java/io/quarkus/scheduler/test/SimpleJobs.java

```diff
@@ -4,8 +4,11 @@
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.CountDownLatch;
 
+import javax.enterprise.context.ApplicationScoped;
+
 import io.quarkus.scheduler.Scheduled;
 
+@ApplicationScoped
```

Comment:
Hm, why did you need to add this?

Reply:
It has to be a bean in order to detect the method annotated with @scheduled

Reply:
But why did you have to add it as part of the PR?

How did it work before?

Reply:
Ah, sorry, my bad. I thought I was watching at SimpleJobs.java in spring-scheduled extension. I should have added it to test something and it slipped through the cracks :-S Let me remove it